docs/usage/security/jwt.rst

```diff
@@ -65,3 +65,32 @@ conversions.
     converting the token. To support more complex conversions, the
     :meth:`~.security.jwt.Token.encode` and :meth:`~.security.jwt.Token.decode` methods
     must be overwritten in the subclass.
+
+
+Verifying issuer and audience
+-----------------------------
+
+To verify the JWT ``iss`` (*issuer*) and ``aud`` (*audience*) claim, a list of accepted
```

Comment:
Should we link to https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3 or some other describer of these terms to help? I was a bit lost

Reply:
I don't think so.  We should probably offer a bigger, more general JWT guide if this is the approach we want to take.

Reply:
Yes. It's my intention to create that together with the (hopefully not too far in the future) upcoming new JWT / security stuff :)